diagrams/create_request_uri.md

```diff
@@ -0,0 +1,49 @@
+```plantuml
+@startuml
+
+autonumber
+
+box "Wallet"
+participant "Metadata" as wm
+participant "Authorization Endpoint" as w
+end box
+
+participant "User Agent" as u
+
+box "Verifier"
+participant "Frontend" as r
+participant "Request Endpoint" as rp
+participant "Response Endpoint" as rb
+end box
+
+u --> r : use
+activate r
+
+r --> u: **signed authorization request**\n(client_id, create_request_uri, state)
+deactivate r
+u --> w: **signed authorization request**\n(client_id, create_request_uri, state)
+activate w
+w --> w: check authorization request signature
```

Comment:
Next steps
w --> ti: check on trustworthiness of r
ti --> w: positive, r is trusted

Reply:
Since the wallet now has the authenticated identity of the verifier, then it can contact its trust infrastructure to ask if the client_id is trusted.
Note that the wallet must have some way of knowing the true binding of the signing key to the client_id, otherwise anyone can sign anyone else's client id using any key pair they have manufactured themself. Maybe the trust infrastructure can confirm this for the wallet, by returning the public key of the client_id, which must match the one on the signed authz request.
Either way, authenticating the request is meaningless if the binding of the key to the client_id is already known

Reply:
right!
Added 

w --> w: check on trustworthiness of Verifier (approach on trust mechanism)

as the data for checking trustworthiness might also be included in the request